bridgerelayer/cmd/types/message.go

```diff
@@ -1,5 +1,19 @@
 package types
 
+// Message contains RLP-encoded transaction data and transaction verification data
+type Message struct {
+	Contents         []byte      // RLP-encoded transaction data
+	VerificationData interface{} // Data used to verify the transaction by the verification module
```

Comment:
This structure looks good for now, and unblocks on me the Substrate follow-thru. We'll probably circle back in subsequent iterations to refine our data structures some more.